libraries/typings-generator/package.json

```diff
@@ -27,7 +27,7 @@
     "@microsoft/rush-stack-compiler-3.5": "workspace:*",
     "@microsoft/node-library-build": "workspace:*",
     "@rushstack/eslint-config": "workspace:*",
-    "@rushstack/heft": "workspace:*",
+    "@rushstack/heft": "0.8.0",
```

Comment:
Hmm. With all the changes in heft, we may not want to pin to a specific version. Maybe use a ^ version? Or "latest" if supported?

Reply:
I agree with you, but this seems to be what other projects are using that have a circular dependency on Heft. Since it is only being used to build, and since build works, I think I will leave this as is for now. What we should do, however, is make a PR that updates all the projects currently set to "@rushstack/heft": "0.8.0". I think that should happen in a separate PR.